src/cp.js

```diff
@@ -54,14 +53,12 @@ function copyFileSync(srcFile, destFile, options) {
     }
   // recursive is a special case for non-files that means "create an equivalent at the dest" rather than reading and writing
   } else if (options.recursive && (srcFileStat.isFIFO() || srcFileStat.isCharacterDevice() || srcFileStat.isBlockDevice())) {
-    if (srcFileStat.isFIFO() ? mkfifo(destFile) : mknod(destFile, srcFileStat)) {
-      fs.chmodSync(destFile, srcFileStat.mode);
-      try {
-        fs.chownSync(destFile, srcFileStat.uid, srcFileStat.gid);
-      } catch (e) {
-        common.error('copyFileSync: could not set ownership for dest file (' + srcFile + ')');
-      }
-    }
+    var type = 'unknown file';
+    if (srcFileStat.isFIFO()) type = 'fifo';
+    if (srcFileStat.isCharacterDevice()) type = 'block device';
+    if (srcFileStat.isBlockDevice()) type = 'character device';
+    common.error('copyFileSync: ' + type + ' is not supported (' + srcFile + ')', { continue: true });
+    common.logLater('copyFileSync: Special files were encountered during this operation. Please investigate shelljs plugins if you would like to add support for these.');
```

Comment:
I think the `common.error()` log message is sufficient. Let's get rid of `logLater` and move something like this into the docs. You can edit the `//@ ` section in this file, and then run `npm run gendocs`.